consoles/console.pm

```diff
@@ -52,9 +52,9 @@ sub screen {
 
 # helper function
 sub sshCommand {
-    my ($self, $host, $gui) = @_;
+    my ($self, $username, $host, $gui) = @_;
 
-    my $sshopts = "-o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -o PubkeyAuthentication=no root\@$host";
+    my $sshopts = "-o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -o PubkeyAuthentication=no $username\@$host";
```

Comment:
Fixing all instances of `sshCommand` is good but making sure `$username` defaults to `root` - just for being on the safe side - is better in my mind.

Reply:
@mnowaksuse xD Sorry

Reply:
as the username is the new first argument, it's a bit hard to leave it out